workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/GameObjectInitializationSystem.cs

```diff
@@ -48,41 +42,25 @@ protected override void OnCreateManager(int capacity)
 
             worker = Worker.GetWorkerFromWorld(World);
             viewCommandBuffer = new ViewCommandBuffer(EntityManager, worker.LogDispatcher);
-            entityGameObjectCreator = new EntityGameObjectCreator(World);
-            entityGameObjectLinker = new EntityGameObjectLinker(World, worker.LogDispatcher);
+            entityManager = World.GetOrCreateManager<EntityManager>();
+            entityGameObjectLinker = World.GetOrCreateManager<EntityGameObjectLinkerSystem>().Linker;
```

Comment:
Here the idea was to give users a good way to access it if they want to build their own linking logic on top of it, putting it in a sensible place in case they don't want to actually use our GameObjectInitializationSystem at all. Not really appropriate to put the EntityGameObjectCreator in the same place because 1) that's coupled to the GameObjectInitializationSystem workflow, 2) that's one static singleton for all workers as it is now.